src/connector/src/sink/remote.rs

```diff
@@ -70,8 +72,8 @@ use crate::sink::{
 macro_rules! def_remote_sink {
     () => {
         def_remote_sink! {
-            { ElasticSearch, ElasticSearchSink, "elasticsearch" }
-            { Opensearch, OpenSearchSink, "opensearch"}
+            { ElasticSearch, ElasticSearchSink, "elasticsearch_v1" }
```

Comment:
To reduce ambiguity, we may rename the variant name to `XxxJava`, and remove the `Rust` suffix in the `XxxRust` variant defined in `SinkImpl`.